docs/docs/frigate/installation.md

```diff
@@ -25,8 +25,8 @@ Frigate uses the following locations for read/write operations in the container.
 - `/media/frigate/clips`: Used for snapshot storage. In the future, it will likely be renamed from `clips` to `snapshots`. The file structure here cannot be modified and isn't intended to be browsed or managed manually.
 - `/media/frigate/recordings`: Internal system storage for recording segments. The file structure here cannot be modified and isn't intended to be browsed or managed manually.
 - `/media/frigate/exports`: Storage for clips and timelapses that have been exported via the WebUI or API.
-- `/tmp/cache`: Cache location for recording segments. Initial recordings are written here before being checked and converted to mp4 and moved to the recordings folder.
-- `/dev/shm`: It is not recommended to modify this directory or map it with docker. This is the location for raw decoded frames in shared memory and it's size is impacted by the `shm-size` calculations below.
+- `/tmp/cache`: Cache location for recording segments. Initial recordings are written here before being checked and converted to mp4 and moved to the recordings folder. Segments requested for download are also concatenated and processed here. It is highly recommended to use a [`tmpfs`](https://en.wikipedia.org/wiki/Tmpfs) memory drive that can fit recorded segments for all your cameras plus the maximum size of video files you intend to download.
```

Comment:
```suggestion
- `/tmp/cache`: Cache location for recording segments. Initial recordings are written here before being checked and converted to mp4 and moved to the recordings folder. Segments requested for download are also concatenated and processed here.
```

Reply:
I think this will confuse users, it is just using RAM after all.

I think this would be more clear under a general "Minimum Recommended RAM" section in the recommended hardware docs

Reply:
I don't think so, because when I read it, I skipped the example compose file, so I did not have a memory drive there until today. I did not understand the importance. It needs to be more explicit for individuals with "varying levels of expertise" such as myself... :wink: 

Can we find a rephrasing of sorts that is acceptable? (See latest commit)